.github/workflows/pr-all-services.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       # This block should eventually become use lando/actions-hyperdrive@v2
       - name: Verify Docker dependencies
         run: |
-          docker --version | grep "20.10."
+          docker --version | grep "23.0."
```

Comment:
we'll need to find a better way to do this eventually - as the actions image updates, this check fails often (currently on 24.0.x)